examples/vector_search.ipynb

```diff
@@ -15,7 +15,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install superduperdb"
+    "!pip install superduperdb==0.0.12\n",
```

Comment:
If hardcoded versioning is needed, please consider `>=`

Also, there is a '\n'. Is it needed ?